src/zjs_ble.c

```diff
@@ -581,9 +581,12 @@ static struct bt_conn_auth_cb zjs_ble_auth_cb_display = {
 
 void zjs_ble_emit_powered_event()
 {
-    const char state[] = "poweredOn";
-    zjs_defer_emit_event(ble_handle->ble_obj, "stateChange", state,
-                         sizeof(state), string_arg, zjs_release_args);
+    // Don't emit if bluetooth hasn't had init called yet
+    if (ble_handle) {
+        const char state[] = "poweredOn";
+        zjs_defer_emit_event(ble_handle->ble_obj, "stateChange", state,
+                             sizeof(state), string_arg, zjs_release_args);
```

Comment:
Will it be emitted later if we don't do it now?

Reply:
No it won't.   But it also doesn't work now either, so this at least fixes the error.  Should we emit when the bluetooth handle is created? Or have the bluetooth module keep track of whether this has been emitted yet?

Reply:
I think this needs to be fixed more fully then so let's leave it open until @jimmy-huang gets back, or we investigate further.